config/spark/spec.json

```diff
@@ -0,0 +1,98 @@
+{
+	"name": "FuseNetwork",
+	"engine": {
+		"authorityRound": {
+			"params": {
+				"stepDuration": "5",
```

Comment:
That's the contract you deployed, right?
What's the cycle length for Consensus?

Reply:
No not yet haven't deployed so the address are subject to change :)